### PR TITLE
Add template "DocuSeal"

### PR DESCRIPTION
**Pull request by vinumweb**
Adding template for DocuSeal as per request: https://discord.com/channels/949251825511575562/1207449212485373952/1207449212485373952

### CODE REVIEW

File: templates/docuseal/index.ts
Comment (tobiasreichel) [inline templates/docuseal/index.ts:1]: Please add 

        "mounts": [
          {
            "type": "volume",
            "name": "data",
            "mountPath": "/data"

**Review by tobiasreichel — CHANGES_REQUESTED** [review]
Add a Volume Mount Point a DocuSeal saves data to disk

**Comment by marktopper** [comment]
Works like a charm. Great job

**Comment by marktopper** [comment]
@deiucanta I can vouch that this template works.

**Comment by serban-alexandru** [comment]
@deiucanta Ready for merge

**Pull request merged**